src/components/EditText.tsx

```diff
@@ -64,8 +64,8 @@ class EditText extends React.Component<IEditTextProps, IEditTextState> {
     });
   }
 
-  textChange(IEditTextEvent: IEditTextEvent) {
-    this.setState({ value: IEditTextEvent.target.value });
+  textChange(e: IEditTextEvent) {
```

Comment:
As I said before, you might want to use full names instead of shorter ones. It carries more meaning